src/main.tf

```diff
@@ -51,6 +60,10 @@ resource "aws_eks_node_group" "node_group" {
     ]
   }
 
+  # adding tags will cause a provisioning error!
+  # don't add tags here
+  # tags = var.md_metadata.default_tags
```

Comment:
> adding tags will cause a provisioning error!

Interesting. Do we know why? I think we'll be using tags for usage-based billing so at some point this will be important to figure out.

Reply:
No idea, just saw the error when testing. 🤷 

Also... in AWS you can set the tags on the _provider_. We do this and it should set the tags on all resources. However, BC gives an error that we aren't setting it on this resource. It's a chicken and egg and tag problem.